src/components/Shared/Filters/NullState/NullState.tsx

```diff
@@ -14,6 +14,7 @@ import {
   TaskFilterSetInput,
```

Comment:
Null State for Excluded contacts should have a different message and no buttons.
<img width="1316" alt="Screenshot 2024-08-21 at 12 55 20 PM" src="https://github.com/user-attachments/assets/d736de5a-5db3-4f92-b888-e5dcb8161f9c">
<img width="1319" alt="Screenshot 2024-08-21 at 12 55 33 PM" src="https://github.com/user-attachments/assets/296d6d58-5cf9-435b-b186-d22b87b6ea00">

Reply:
Fixed in future PR